telegraf/entrypoint-shim.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+jinja2 /telegraf.conf.j2 > "/etc/telegraf/telegraf.conf"
+
+if [ "${1:0:1}" = '-' ]; then
```

Comment:
this file needs some comments, it is not clear what is happening

Reply:
as for the host variable... need to look above the TELEGRAF variables.